clicommand/agent_start.go

```diff
@@ -165,6 +165,7 @@ type AgentStartConfig struct {
 	Experiments       []string `cli:"experiment" normalize:"list"`
 	Profile           string   `cli:"profile"`
 	StrictSingleHooks bool     `cli:"strict-single-hooks"`
+	KuberentesExec    bool     `cli:"kubernetes-exec"`
```

Comment:
Heads up there's a typo here Kuber`en`tesExec

Reply:
lol, whoops. i've just put a PR in to fix it, but there should be no impact